translations/locales/hi/translations.json

```diff
@@ -109,7 +109,6 @@
       "PrivacyPolicy": "गोपनीयता नीति",
       "TermsOfUse": "उपयोग की शर्तें",
       "CodeOfConduct": "आचार संहिता"
```

Comment:
I think you should be fixing it by adding a `,` here, thereby making it a valid JSON.